lesson03/assignment/scripts/main.js

```diff
@@ -0,0 +1,48 @@
+//pseudo code outline for script
+
+//get json data entries:
+$.getJSON('../data/data.json', function(data) {
+  // console.log(data);
+/* since '$.getJSON' is asynchronous, we need to call a helper function within
+  the callback so that there wont be any issues with using the date we're getting 
+  from data.json */
+  createCards(data);
+});
+
+//we make the helper function here.
+function createCards(data) {
+  let dataHTML = data.map(function(el) {
+    //i know this isn't typical html insertion but this feels much more simplier than adding "appends" everywhere.
+    let card = 
```

Comment:
While we don't expect students to use this solution, this is certainly the cleanest way of doing it! Props to you for going above and beyond on this one.